server/report/models.py

```diff
@@ -1,14 +1,14 @@
 from django.db import models
 import uuid
+from interview.models import Interview
+from questions.models import TechnicalQuestion
+from members.models import User
 
-from django.contrib.auth.models import User
-
-# Create your models here.
 class Report(models.Model):
     REPORT_TYPE_CHOICES = [
         ('interview', 'Interview'),
         ('question', 'Question'),
-        # Add other types as needed
+        ('member', 'Member'),
```

Comment:
Why are we passing tuples why not just make it 'member' etc.

Reply:
(DB, presentation)

this is the syntax for choices fields iiuc